src/main/java/frc/robot/swerve/SwerveConstants.java

```diff
@@ -7,43 +7,51 @@
 
 public class SwerveConstants {
     public static final double[] OFFSETS =
-            {0.5686593642164841, 0.26232458155811456, 0.00854002521350063, 0.5429330635733266};
+            {0.6002859900071498,0.5906595147664879,0.3208662830216571,0.19138357978458953};
```

Comment:
These shouldn't be here, these are different for each robot and should be supplied as a parameter by the user in their repo.

Reply:
Maybe we should change this after we merge the phoenix and this pr?

Reply:
It'll be much easier then